fulp_modules/main_features/mentorhelp/mentorsay.dm

```diff
@@ -12,7 +12,9 @@
 
 	log_mentor("MSAY: [key_name(src)] : [msg]")
 	msg = keywords_lookup(msg)
-	if(mentor_datum?.is_contributor)
+	if(src.key == "SgtHunk")
```

Comment:
```suggestion
	if(src.key == "johnwillard")
```

yeah sorry about this sudden retirement, a shame too

Reply:
its so sad John Willard was found dead with 5 gunshots to his head
i didnt know suicide could be like that